server/src/logic.js

```diff
@@ -335,6 +336,12 @@ export const getHandlers = ({ models, creators: Creators, push }) => {
       eventLocations(event) {
         return event.getEventlocations();
       },
+      primaryLocation(event) {
+        // to avoid a cyclic table associations we are calling 'constraints: false'
```

Comment:
maybe mention this is in the model definition:
```
// to avoid a cyclic table association we create the EventModel with `constraints: false`
```

Reply:
already mentioned.